src/axolotl/utils/schemas/config.py

```diff
@@ -855,7 +855,7 @@ class AxolotlInputConfig(
         },
     )
     include_tkps: bool | None = Field(
-        default=None,
+        default=True,
         json_schema_extra={
             "description": "bool of whether to report tokens per second during training by measuring throughput of non-padding tokens."
```

Comment:
I didn't get to ask, is this tkps for all GPUS/across nodes or only single GPU? Should we add a small point on that here?